sled-agent/src/bootstrap/sprockets_server.rs

```diff
@@ -59,22 +59,33 @@ impl SprocketsServer {
     /// which is cancel-safe. Note that cancelling this
     /// server does not necessarily cancel any outstanding requests that it has
     /// already received (and which may still be executing).
-    pub(super) async fn run(mut self) {
+    pub(super) async fn run(self) {
         loop {
             // Sprockets actually _uses_ the key here!
-            let (stream, remote_addr) = match self.listener.accept().await {
-                Ok(conn) => conn,
+            // TODO: Once we have a corpus, use it.
+            // Will we ever have one at RSS time?
```

Comment:
I believe we should because the expectation is that sleds have gotten a mupdate but we should review this